src/issue/screens/new-issue.screen.js

```diff
@@ -0,0 +1,164 @@
+
+const mapDispatchToProps = dispatch => ({
+  submitNewIssueByDispatch: (
+    owner,
+    repoName,
+    issueTitle,
+    issueComment
+  ) =>
+    dispatch(
+      submitNewIssue(
+        owner,
+        repoName,
+        issueTitle,
+        issueComment
+      )
+    ),
+});
+
+class NewIssue extends Component {
+  props: {
+    submitNewIssueByDispatch: Function,
+    repository: Object,
+    navigation: Object,
+  };
+
+  state: {
+    issueTitle: string,
+    issueComment: string,
+  };
+
+  constructor() {
+    super();
+
+    this.state = {
+      issueTitle: '',
+      issueTitleHeight: 0,
+      issueComment: '',
+      issueCommentHeight: 0,
+    };
+  }
+
+  submitNewIssue = () => {
+    const {
+      submitNewIssueByDispatch,
+      repository,
+      navigation,
+    } = this.props;
+    const { issueTitle, issueComment } = this.state;
+    const repoName = repository.name;
+    const owner = repository.owner.login;
+
+    if (issueTitle === '') {
+      Alert.alert('You need to have an issue title!', null, [{ text: 'OK' }]);
+    } else {
+      submitNewIssueByDispatch(
+        owner,
+        repoName,
+        issueTitle,
+        issueComment
+      ).then(issue => {
+        navigation.navigate('Issue', {
+          issue,
+        });
+      });
+    }
+  };
+
+  render() {
+    const { issueTitle, issueComment } = this.state;
+
+    return (
+      <ViewContainer>
+        <ScrollView>
+          <SectionList title="Issue Title">
+            <TextInput
+              underlineColorAndroid={'transparent'}
+              placeholder="Write a title for your issue here"
+              blurOnSubmit
+              onContentSizeChange={event =>
+                this.setState({
+                  issueTitleHeight: event.nativeEvent.contentSize.height,
+                })}
+              onChangeText={text => this.setState({ issueTitle: text })}
+              placeholderTextColor={colors.grey}
+              style={[
+                styles.textInput,
+                { height: Math.max(60, this.state.issueTitleHeight) },
+              ]}
+              value={issueTitle}
+            />
+          </SectionList>
+
+          <SectionList title="Issue comment">
+            <TextInput
+              underlineColorAndroid={'transparent'}
+              placeholder="Write a comment for your issue here"
+              blurOnSubmit
+              multiline
+              onChangeText={text => this.setState({ issueComment: text })}
+              onContentSizeChange={event =>
+                this.setState({
+                  issueCommentHeight: event.nativeEvent.contentSize.height,
+                })}
+              placeholderTextColor={colors.grey}
+              style={[
+                styles.textInput,
+                { height: Math.max(60, this.state.issueCommentHeight) },
+              ]}
+              value={issueComment}
+            />
+          </SectionList>
+
+          <SectionList title="Create new issue">
```

Comment:
I think the "Submit" button is pretty self-explanatory, so the "Create New Issue" text doesn't need to be repeated.